benchmarking/src/lib.rs

```diff
@@ -662,27 +680,39 @@ macro_rules! impl_benchmark {
 		{ $( $instance:ident )? }
 		( $( { $( $name_inst:ident )? } $name:ident )* )
 		( $( $name_extra:ident ),* )
+		( $( $name_skip_meta:ident ),* )
 	) => {
 		pub struct Benchmark;
 
 		impl $crate::Benchmarking<$crate::BenchmarkResults> for Benchmark {
-			fn benchmarks(extra: bool) -> $crate::Vec<&'static [u8]> {
-				let mut all = $crate::vec![ $( stringify!($name).as_ref() ),* ];
+			fn benchmarks(extra: bool) -> $crate::Vec<$crate::BenchmarkMetadata> {
```

Comment:
this is one of the changes in benchmarking, and we may need to check latest `frame-branchmarking` impl, to follow the improvements and updates. then redo benches before next runtime release